.github/workflows/stale.yml

```diff
@@ -2,8 +2,14 @@ name: 'Close stale issues and PRs'
 on:
   schedule:
     - cron: '0 15 * * *'
+permissions:
```

Comment:
Is this needed? 🤔 The `stale` Action only needs access to `issues` and `pull-requests`